README.adoc

```diff
@@ -1,91 +1,99 @@
-    -se | --schedulesEnabled - installs scheduling infrastructure and configures SCDF to use the service.
-    -na | --noAutoreconfiguration - tell the buildpack to disable spring autoreconfiguration
-
-[*] = Required arguments
-```
-
-The first option is to choose a *PLATFORM*, available options `cloudfoundry`, `local`, `gke` (kubernetes) and `pks` (kubernetes).  The scripts for each platform are located in folders of the same name in the main directory.
-
-By default the script will execute three main phases:
+* Setup: The link:scripts/setup.adoc[setup] phase will traverse each folder and call `create.sh` scripts.
+At the end of this phase you should expect to have an environment available with the Spring Cloud Data Flow server with a verified `SERVER_URI` and `SKIPPER_SERVER_URI`, along with the services required for it to run.
+* Test: The link:scripts/tests.adoc[test] phase will invoke the `mvn test`. The Junit test fixtures deploy test apps, using the `DataFlowTemplate`, into the configured environment and verify that the streams and tasks complete as expected.
+* Clean: The link:scripts/clean.adoc[clean] phase will undeploy the server and remove any services.
 
-* Setup: The setup phase will traverse each folder and call `create.sh` scripts.  At the end of this phase you should expect to have an environment available with the Spring Cloud Data Flow server along with the services required for it to run.
-* Test: The test phase will invoke the `mvn test` and deploy apps into the environment and run tests.
-* Clean: The clean up phase will undeploy the server and remove any services.
+NOTE: For convenience, `test` includes `clean` after by default. You can set `--skipClean` to disable this.
 
-Each phase can be toggled by setting the appropriate flag (-s, -t, -c)
+```
+USAGE: run.sh <phase> <phase-args>
+```
 
-The services created in the setup phase are `mysql` and `redis`.  Depending on the binder selected, it will create a Rabbit or Kafka service.
+The first option is to choose a *PLATFORM*, available options `cloudfoundry`, `kubernetes`, and `local`.
+The scripts specific to each platform are located in directories named for the corresponding platform.
 
 == Examples
 
-To run the tests locally cleaning up services using Rabbit (the default binder)
-
-`./run.sh -p local`
-
-To run the tests locally cleaning up services using Kafka
-
-`./run.sh -p local -b kafka`
-
-To run the tests locally and keep the Data Flow Server, Kafka, and other services running afterwards
-
-`./run.sh -p local -b kafka -c`
+=== To run the tests on kubernetes:
 
+```
+#The kubernetes config location, e.g., ~/.kube/config
+export KUBECONFIG=config location
+export CLUSTER_NAME=<cluster-name>
+#By default the setup will perform a helm install. Uncomment to install directly from the git repo.
+#export USE_DISTRO_FILES=true
+
+run.sh clean -p kubernetes -se #schedules enabled
```

Comment:
since it looks like the only platform that doesn't have scheduling enabled is local, should we default to schedules enabled and have local disable it? i can't recall a time where we have ever disabled it, for k8s at least

Reply:
cloudfoundry is a bit flaky with schedules IIUC @cppwfs  ?